packages/elements/src/components/ino-accordion/ino-accordion.scss

```diff
@@ -35,7 +40,7 @@ $animation-time: 200ms;
   }
 
   &--expanded {
-    background: theme.$p-1;
+    background: var(--inovex-elements-p-1); //Example for CSS variable usage
```

Comment:
This is a great start for switching between light and dark theme!
The approach of using a second theme-color-pallet with the same names as the given light-theme is great, because we won't need to introduce more variables.

But i have the concern, that we won't have a dark theme-color-pallet which will perfectly match with the light-theme colors. 

Because some components might work which their light-theme colors in dark mode but some might need bigger adjustments which won't perfectly fit across all other components. 

If we won't find a fitting dark-theme-color-pallet I would suggest to store the component colors as variable (e.g. `$ino-accordion-bg-color`) which will choose, according to the selected theme, what color should be displayed. This will allow a fine granular adjustment of all colors.

Example: 
**`ino-accordion-colors.scss`**(holds only accordion colors) or **`ino-components-colors.scss`**(holds colors of every component)
````
.inovex-elements-theme-light {
 $ino-accordion-bg-color: theme.$p-2;
 $ino-accordion-icon-color: theme.$p-5;
 $ino-accordion-text-color: theme.$n-11;
 ...
}

.inovex-elements-theme-dark {
 $ino-accordion-bg-color: theme.$n-8;
 $ino-accordion-icon-color: theme.$p-5;
 $ino-accordion-text-color: theme.$white;
 ...
}
````
But with smaller deviation we won't need this procedure.

Waiting for #1058 and dark-theme-pallet